.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: generic
+services:
+  - docker
+env:
+  global:
+    - HUB_REPO=herbrandson/k8dash
+    - BINFMT_TAG=a7996909642ee92942dcd6cff44b9b95f08dad64
+    - COMMIT=${TRAVIS_COMMIT::8}
+    - DOCKER_CLI_EXPERIMENTAL=enabled
+cache:
+  directories:
+    - docker_images
+    - $HOME/.cache/trivy
+before_cache:
+  - docker save -o docker_images/images.tar $(docker images -a -q)
+before_install:
+  - sudo apt-get -y update || true
+  - sudo apt-get -y install qemu-system-arm qemu-user-static binfmt-support
+  - docker run --rm --privileged docker/binfmt:${BINFMT_TAG}
+  - sudo update-binfmts --display
+  - curl -fsSL https://get.docker.com -o get-docker.sh
+  - sudo sh get-docker.sh
+  - echo ${HUB_TOKEN} | docker login --username ${HUB_USER} --password-stdin
+  - docker load -i docker_images/images.tar || true
+  - docker buildx create --use
+  - docker buildx build ./ --platform=linux/arm/v6,linux/arm/v7,linux/arm64/v8,linux/amd64,linux/i386 -t ${HUB_REPO}:${COMMIT} -t ${HUB_REPO}:latest --push
+  - docker save -o docker_images/images.tar $(docker images -a -q)
+  - export VERSION=$(curl --silent "https://api.github.com/repos/aquasecurity/trivy/releases/latest" | grep '"tag_name":' | sed -E 's/.*"v([^"]+)".*/\1/')
+  - wget https://github.com/aquasecurity/trivy/releases/download/v${VERSION}/trivy_${VERSION}_Linux-64bit.tar.gz
```

Comment:
Just curious... any reason you're using wget here and curl above?

Reply:
Copy & Paste from the individual snippets :))